### PR TITLE
Update bigquery library version, it's over a year old.

### PR DESCRIPTION
**Pull request by vimota**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -292,10 +292,7 @@ RUN pip install --upgrade cython && \
     # Need to use CountEncoder from category_encoders before it's officially released
     pip install git+https://github.com/scikit-learn-contrib/categorical-encoding.git && \
     pip install google-cloud-automl && \
-    # Newer version crashes (latest = 1.14.0) when running tensorflow.
```
Comment (vimota) [inline Dockerfile:295]: Added a test to check this

**Comment by vimota** [comment]
Currently blocked on https://b.corp.google.com/issues/168130817

**Comment by rosbo** [comment]
@vimota I fixed the build yesterday and `master` is green.

**Comment by vimota** [comment]
> @vimota I fixed the build yesterday and `master` is green.

Thank you!!

**Comment by rosbo** [comment]
Closing in favor of #963

**Pull request closed without merging**